hedera-node/hedera-app-spi/src/testFixtures/java/com/hedera/node/app/spi/fixtures/workflows/FakePreHandleContext.java

```diff
@@ -188,9 +188,10 @@ public Set<Account> optionalHollowAccounts() {
     @Override
     public PreHandleContext optionalSignatureForHollowAccount(@NonNull final Account hollowAccount) {
         requireNonNull(hollowAccount);
+        final AccountID accountID = hollowAccount.accountId();
         if (!isHollow(hollowAccount)) {
-            throw new IllegalArgumentException(
-                    "Account %d is not a hollow account".formatted(hollowAccount.accountNumber()));
+            throw new IllegalArgumentException("Account %d.%d.%d is not a hollow account"
```

Comment:
This is another thing that should go into a "foundation" area - why isn't there a `ToString()` somewhere that will do this formatting? We also see this in `PrehandleContextImpl`.  

We need a place for mono's `EntityIdUtils` - and other common utilities for foundation types.